helm_deploy/templates/custom-error-pages.yaml

```diff
@@ -0,0 +1,107 @@
m){.app-content .app-video__description{margin-bottom:30px}}.app-content pre,.app-content code{font-family:monospace,monospace;font-size:1em}.app-example-page__wrapper{overflow:auto;background-color:#fff}.app-example-page{padding:30px}.app-copy-button{font-family:"GDS Transport",arial,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;font-weight:400;font-size:14px;font-size:.875rem;line-height:1.1428571429;position:absolute;z-index:1;top:10px;right:10px;min-width:110px;padding:3px 10px;border:1px solid #00823b;color:#00823b;background-color:#fff;-webkit-box-shadow:0 2px 0 0 #00703c;box-shadow:0 2px 0 0 #00703c;text-align:center;text-decoration:none;cursor:pointer}@media print{.app-copy-button{font-family:sans-serif}}@media(min-width: 40.0625em){.app-copy-button{font-size:16px;font-size:1rem;line-height:1.25}}@media print{.app-copy-button{font-size:14pt;line-height:1.2}}.app-copy-button:focus:not(:hover){color:#0b0c0c;background-color:#fd0;-webkit-box-shadow:0 2px 0 0 #0b0c0c;box-shadow:0 2px 0 0 #0b0c0c}.app-copy-button:active,.app-copy-button:focus{padding:2px 10px;border:2px solid #fd0;outline:2px solid rgba(0,0,0,0);-webkit-box-shadow:none;box-shadow:none}.app-copy-button:active{margin-top:2px;-webkit-box-shadow:none;box-shadow:none}.app-colour-list{width:100%;border-collapse:collapse}.app-colour-list-row{display:table-row;margin-bottom:10px;border-bottom:10px solid rgba(0,0,0,0)}@media(min-width: 61.25em){.app-colour-list-row{display:table-row;margin:auto;border-bottom:0}}.app-colour-list-column{display:block;position:relative;padding-left:50px}@media(min-width: 61.25em){.app-colour-list-column{display:table-cell;padding-left:0;vertical-align:middle}}.app-swatch{display:block;position:absolute;top:0;left:0;width:40px;height:40px;border:1px solid rgba(0,0,0,0);border-radius:50%}@media(min-width: 61.25em){.app-swatch{display:inline-block;position:static;margin-top:6px;margin-right:10px;margin-bottom:7px;vertical-align:middle}}.app-swatch-border{border-color:#b1b4b6}.app-colour-list-column--name{font-weight:normal;text-align:left}@media(min-width: 61.25em){.app-colour-list-column--name{width:50%}}@media(min-width: 61.25em){.app-colour-list-column--colour{width:15%}}.app-colour-list-column--colour code{padding:0;color:#0b0c0c;background:none}@media(min-width: 61.25em){.app-colour-list-column--notes{width:35%}}pre{max-width:38em;margin:0;padding:20px;overflow:auto;border:1px solid #b1b4b6;background-color:#f8f8f8;font-size:16px;margin-bottom:15px}@media(min-width: 40.0625em){pre{margin-bottom:20px}}li code,td code,p code{padding:0 5px;color:#d13118;background-color:#f3f2f1}.app-prose-scope strong,.app-prose-scope b{font-weight:700}.app-prose-scope a:focus code{color:#0b0c0c;background:rgba(0,0,0,0)}.app-prose-scope pre+h2{padding-top:20px}.app-prose-scope pre+h3,.app-prose-scope pre+h4{padding-top:10px}.app-prose-scope>:last-child{margin-bottom:0}.app-whats-new{border-bottom:1px solid #b1b4b6;background-color:#f8f8f8}
```

Comment:
I don't like this....

it will be difficult to maintain if/when the designs change....

options (bad):
* turning this into a erb and rendering the page as part of the deploy 
* something else